guillotina_kafka/commands/kafka_multi_consumer.py

```diff
@@ -67,14 +71,16 @@ def get_worker(self, name):
 
         if inspect.isclass(worker):
             worker = worker()
-
+        worker.__consumer__ = consumer
```

Comment:
Maybe we should add a comment to remember why we are doing this.

Reply:
done!